VEDA_OS01/management/commands/re_encrypt_transcript_credentials.py

```diff
@@ -0,0 +1,43 @@
+"""
+Management command used to re-encrypt transcript credentials data with new fernet key.
+"""
+import logging
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from cryptography.fernet import InvalidToken
+
+from VEDA_OS01.models import TranscriptCredentials
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Re-encrypt trancript credentials command class.
+    """
+    help = 'Re-encrypts transcript credentials with new fernet key.'
+
+    def handle(self, *args, **options):
+        """
+        handle method for command class.
+        """
+
+        LOGGER.info('[Transcript credentials re-encryption] Process started.')
+
+        try:
+            # Get all objects
+            transcript_credentials = TranscriptCredentials.objects.all()
+
+            with transaction.atomic():
+                # Call save on each credentials record so that re-encryption can be be performed on fernet fields.
+                for transcript_credential in transcript_credentials:
```

Comment:
I think `for transcript_credential in TranscriptCredentials.objects.all():` is enough, no need to create separate variable above.